### ISSUE TITLE
CSS import rule throws

### ISSUE THREAD

**Issue by DrixnFS** [yWorks/svg2pdf.js#177]
I have the following problem:
Hello community, iam trying to find a library that will allow me to convert my custom made svgs into PDFs and stay svg and i went by this. I tried the online demo and tried to implement it in my project but both gives me the same error of : Cannot read property 'indexOf' of undefined inside a  stylesheets.ts:76 so its internall error. I tried sanitazing the SVG and it does the same. In every matter it should be valid SVG but its huge one

![image](https://user-images.githubusercontent.com/52955276/118184481-b3f24000-b43b-11eb-81ac-af420e5cdd51.png)
[svg.txt](https://github.com/yWorks/svg2pdf.js/files/6474932/svg.txt)
Basically this is the svg as picture what iam trying to get. and also adding the source of the svg if anyone would have any idea how to workaround the issue or what exactly iam missing it would mean the world to me Thanks alot community

**Comment by yGuy** [comment]
The problem is the @import in the CSS which is not supported. We expect a selector text but this is undefined.
Removing the @import gets rid of the exception.
[Small test-case](http://raw.githack.com/yWorks/svg2pdf.js/master/?svg=%253Csvg%2520font-family%253D%2522%27Oswald%27%252Csans-serif%2522%2520xmlns%253D%2522http%253A%252F%252Fwww.w3.org%252F2000%252Fsvg%2522%253E%253Cstyle%2520xmlns%253D%2522http%253A%252F%252Fwww.w3.org%252F1999%252Fxhtml%2522%253E%2540import%2520url%28%2522https%253A%252F%252Ffonts.googleapis.com%252Fcss2%253Ffamily%253DOswald%253Awght%2540200%2526amp%253Bdisplay%253Dswap%2522%29%253B%253C%252Fstyle%253E%253C%252Fsvg%253E)

**Comment by lucaslar** [comment]
Hello there, I'm facing the same error even though I don't have any `@import`. Instead, `@font-face` causes the problem &rarr; if I take it out, it works, but, obviously, the PDF is not looking as good as it should. Is this intended to be fixed, i.e. are external ressources automatically to be inlined on generating a PDF?

In my case, I'm "printing" a current part of the screen (dom &rarr; svg &rarr; pdf - the latter step using this package). For the first step, I found another really good package which also allows to inline external resources ([dom-to-svg](https://github.com/felixfbecker/dom-to-svg)). @DrixnFS Perhaps this works for you, too!
